drivers/display/display_sdl_bottom.c

```diff
@@ -136,6 +188,14 @@ void sdl_display_write_bottom(const uint16_t height, const uint16_t width, const
 				       color_tint & 0xff);
 		SDL_RenderCopy(renderer, texture, NULL, NULL);
 		SDL_SetTextureColorMod(texture, 255, 255, 255);
+
+		/* Apply ellipse mask if enabled */
+		if (rounded_mask && mask_texture != NULL) {
```

Comment:
nit: it would be possible to just do (mask_texture != NULL) and save passing rounded_mask and the first part of the check.
(same in sdl_display_blanking_off_bottom)